DQM/L1TMonitor/src/L1TObjectsTiming.cc

```diff
@@ -0,0 +1,480 @@
+            int index = itBX - std::min(0, 1 - (int)bxrange%2 - (int)std::floor(bxrange/2.)); // the correlation from itBX to respective index of the vector
+            muons_eta_phi_isolated.at(index)->Fill(muon->eta(),muon->phi());
+          }
+        }
+        for (int itBX = JetBxCollection->getFirstBX(); itBX <= JetBxCollection->getLastBX(); ++itBX) {
+          for (l1t::JetBxCollection::const_iterator jet = JetBxCollection->begin(itBX); jet != JetBxCollection->end(itBX); ++jet) {
+            int index = itBX - std::min(0, 1 - (int)bxrange%2 - (int)std::floor(bxrange/2.)); // the correlation from itBX to respective index of the vector
+            jet_eta_phi_isolated.at(index)->Fill(jet->eta(), jet->phi());
+          }
+        }
+        for (int itBX = EGammaBxCollection->getFirstBX(); itBX <= EGammaBxCollection->getLastBX(); ++itBX) {
+          for (l1t::EGammaBxCollection::const_iterator egamma = EGammaBxCollection->begin(itBX); egamma != EGammaBxCollection->end(itBX); ++egamma) {
+            int index = itBX - std::min(0, 1 - (int)bxrange%2 - (int)std::floor(bxrange/2.)); // the correlation from itBX to respective index of the vector
+            egamma_eta_phi_isolated.at(index)->Fill(egamma->eta(), egamma->phi());
+          }
+        }
+        for (int itBX = TauBxCollection->getFirstBX(); itBX <= TauBxCollection->getLastBX(); ++itBX) {
+          for (l1t::TauBxCollection::const_iterator tau = TauBxCollection->begin(itBX); tau != TauBxCollection->end(itBX); ++tau) {
+            int index = itBX - std::min(0, 1 - (int)bxrange%2 - (int)std::floor(bxrange/2.)); // the correlation from itBX to respective index of the vector
+            tau_eta_phi_isolated.at(index)->Fill(tau->eta(), tau->phi());
+          }
+        }
+        for (int itBX = EtSumBxCollection->getFirstBX(); itBX <= EtSumBxCollection->getLastBX(); ++itBX) {
+          for (l1t::EtSumBxCollection::const_iterator EtSum = EtSumBxCollection->begin(itBX); EtSum != EtSumBxCollection->end(itBX); ++EtSum) {
+            int index = itBX - std::min(0, 1 - (int)bxrange%2 - (int)std::floor(bxrange/2.)); // the correlation from itBX to respective index of the vector
+            if (l1t::EtSum::EtSumType::kMissingEt == EtSum->getType()) {
+              etsum_eta_phi_MET_isolated.at(index)->Fill(EtSum->phi());
+            }
+            else if (l1t::EtSum::EtSumType::kMissingEtHF == EtSum->getType()) {
+              etsum_eta_phi_METHF_isolated.at(index)->Fill(EtSum->phi());
+            }
+            else if(l1t::EtSum::EtSumType::kMissingHt == EtSum->getType()) {
+              etsum_eta_phi_MHT_isolated.at(index)->Fill(EtSum->phi());
+            }
+            else if(l1t::EtSum::EtSumType::kMissingHtHF == EtSum->getType()) {
+              etsum_eta_phi_MHTHF_isolated.at(index)->Fill(EtSum->phi());
+            }
+          }
+        }
+      }
+      
+      //if(itr->getAlgoDecisionInitial(algoBitFirstBxInTrain_))
```

Comment:
This should not be commented out. It checks if the algo actually fired and that is a requirement to fill the histograms.